### PR TITLE
Guide meta typo

### PR DESCRIPTION
**Pull request by raveling**
make <title> match H1

### CODE REVIEW

**Review by dominikwilkowski — CHANGES_REQUESTED** [review]
We need to use the default way here. Let me write it up in slack

File: .changeset/strong-bobcats-shave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/website': patch
+---
+
+fix typo in guide meta title
```
Comment (timleslie) [inline .changeset/strong-bobcats-shave.md:5]:
```suggestion
Fixed typo in guide meta title.
```

**Review by dominikwilkowski — APPROVED** [review]

File: docs-next/pages/guides/keystone-5-vs-keystone-next.mdx [hunk 2]
```diff
@@ -89,15 +87,6 @@ You’ll have to rewrite your config, schema, session and access control functio
...
-    <Page
-      title={'Should you use Keystone 5 or Keystone Next'}
-      headings={headings}
-      isProse
-    >
-      <MDXProvider components={components}>{children}</MDXProvider>
-    </Page>
-  );
-};
+export default ({ children }) => <Markdown>{children}</Markdown
```
Comment (dominikwilkowski) [inline docs-next/pages/guides/keystone-5-vs-keystone-next.mdx:90]:
```suggestion
export default ({ children }) => <Markdown>{children}</Markdown>
```

File: docs-next/pages/guides/keystone-5-vs-keystone-next.mdx [hunk 3]
```diff
@@ -89,15 +87,6 @@ You’ll have to rewrite your config, schema, session and access control functio
...
-      headings={headings}
-      isProse
-    >
-      <MDXProvider components={components}>{children}</MDXProvider>
-    </Page>
-  );
-};
+export default ({ children }) => <Markdown>{children}</Markdown>
+
+>
```
Comment (dominikwilkowski) [inline docs-next/pages/guides/keystone-5-vs-keystone-next.mdx:92]:
```suggestion

```

**Pull request merged**